examples/near_global_ocean_simulation.jl

```diff
@@ -0,0 +1,268 @@
+# # Near-global Ocean Simulation
+#
+# The example sets up and runs a near-global ocean simulation using the Oceananigans.jl and ClimaOcean.jl packages.
+# The simulation covers latitudes from 75°S to 75°N with a horizontal resolution of 1/4 degree and 40 vertical levels. 
+#
+# The ouptut of the simulation is visualized with the CairoMakie.jl package.
+#
+# ## Initial Setup with Package Imports
+#
+# We begin by importing the necessary Julia packages for visualization (CairoMakie),
+# ocean modeling (Oceananigans, ClimaOcean), and handling dates and times (CFTime, Dates).
+# These packages provide the foundational tools for setting up the simulation environment, 
+# including grid setup, physical processes modeling, and data visualization.
+
+using Oceananigans
+using Oceananigans.Units
+using Oceananigans: architecture, on_architecture
+using ClimaOcean
+using ClimaOcean.ECCO
+using ClimaOcean.OceanSimulations
+using ClimaOcean.OceanSeaIceModels
+using OrthogonalSphericalShellGrids
```

Comment:
Do we need `OrthogonalSphericalShellGrids` here?

Reply:
No here we don't. I was trying to make this a global simulation but it looks like with the `OrthogonalSphericalShellGrid` we incur in parameter space limitations on the calthech cluster GPUs (they are an sm_60), specifically in the flux computation. 

For the moment we can have this example on a lat long grid but my goal is to have the global ocean here.